acitoolkit/acisession.py

```diff
@@ -43,6 +47,8 @@
 import weakref
 from collections import namedtuple
 
+import six
```

Comment:
Should this be moved up to be part of other imports?

Reply:
Will move - thanks!